ui/src/app/applications/components/application-sync-options/application-sync-options.tsx

```diff
@@ -111,9 +113,17 @@ export const ApplicationSyncOptions = (props: ApplicationSyncOptionProps) => (
                 {render(props)}
             </div>
         ))}
-        <div className='small-12' style={optionStyle}>
-            {booleanOption('Replace', 'Replace', false, props, false, REPLACE_WARNING)}
-        </div>
+        <DataLoader
+            load={async () => {
+                const settings = await services.authService.settings();
+                return settings.syncWithReplaceAllowed;
+            }}>
+            {syncWithReplaceAllowed => (
+                <div className='small-12' style={optionStyle}>
+                    {booleanOption('Replace', 'Replace', false, props, false, REPLACE_WARNING, !syncWithReplaceAllowed)}
+                </div>
+            )}
+        </DataLoader>
```

Comment:
If it is disabled globally in the server, do not display it in the UI. There is not value to show it in a disabled mode.